### PR TITLE
add ozone yaml files

### LINKED ISSUE #48: move ozone data to clean bucket

### PR DESCRIPTION
**Pull request by jrknezha**

### CODE REVIEW

File: data-tree/reanalysis/mls/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+archive of mls data
```
Comment (frolovsa) [inline data-tree/reanalysis/mls/README.md]: i think we should have a directory tree like 
data-tree/reanalysis/ozone/nasa/mls/README.md

File: data-tree/reanalysis/mls/nasa/2004-2021.clean_bucket.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    platform: aws_s3
+    key:  observations/scratch_from_jrk/mls/oz_v5/netcdf/Y%Y/M%m/
+    bucket: noaa-reanalyses-pds
+    file_template:
+      prefix: "MLS-v5.0-oz.%Y%m%d_%Hz"
+      suffix: ".nc"
+    credential_profile: rnr-bdp
+destination:
+  platform: aws_s3
+  path: "observations/reanalysis/mls/nasa/mlsv5/%Y/%m/netcdf/"
```
Comment (frolovsa) [inline data-tree/reanalysis/mls/nasa/2004-2021.clean_bucket.yaml]: can we make it ozone/nasa/mls

File: data-tree/reanalysis/omi/README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+archive of omi data
```
Comment (frolovsa) [inline data-tree/reanalysis/omi/README.md]: similar change to a path
reanalysis/ozone/nasa/omi-eff/

File: data-tree/reanalysis/omi/nasa/2004-2020.clean_bucket.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    platform: aws_s3
+    key:  observations/scratch_from_jrk/omi_eff_adjusted/Y%Y/M%m/
+    bucket: noaa-reanalyses-pds
+    file_template:
+      prefix: "OMIeff-adj.%Y%m%d_%Hz"
+      suffix: ".nc"
+    credential_profile: rnr-bdp
+destination:
+  platform: aws_s3
+  path: "observations/reanalysis/omi/nasa/omi_eff_adjusted/%Y/%m/netcdf/"
```
Comment (frolovsa) [inline data-tree/reanalysis/omi/nasa/2004-2020.clean_bucket.yaml]: similar change to a path
reanalysis/ozone/nasa/omi-eff/

I encourage you to read up on each of the data streams that you are moving. I learened a lot when i started curating this bucket. see this link for omi
https://aura.gsfc.nasa.gov/omi.html
If you have questions about what you are reading. lets chat.

File: data-tree/reanalysis/omps/README.md [hunk 5]
```diff
@@ -0,0 +1 @@
+achive of omps data
```
Comment (frolovsa) [inline data-tree/reanalysis/omps/README.md]: reanalysis/ozone/nasa/omps-lp/README.md

File: data-tree/reanalysis/omps/nasa/2012-2021.ompslp.clean_bucket.yaml [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    platform: aws_s3
+    key:  observations/scratch_from_jrk/ompslp/Y%Y/M%m/
+    bucket: noaa-reanalyses-pds
+    file_template:
+      prefix: "OMPS-LPoz-Vis.%Y%m%d_%Hz"
+      suffix: ".nc"
+    credential_profile: rnr-bdp
+destination:
+  platform: aws_s3
+  path: "observations/reanalysis/omps/nasa/ompslp/%Y/%m/netcdf/"
```
Comment (frolovsa) [inline data-tree/reanalysis/omps/nasa/2012-2021.ompslp.clean_bucket.yaml]: reanalysis/ozone/nasa/omps-lp

File: data-tree/reanalysis/omps/nasa/2012-2021.ompsnm_eff.clean_bucket.yaml [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    platform: aws_s3
+    key:  observations/scratch_from_jrk/ompsnm_eff/Y%Y/M%m/
+    bucket: noaa-reanalyses-pds
+    file_template:
+      prefix: "OMPSNM.%Y%m%d_%Hz"
+      suffix: ".nc"
+    credential_profile: rnr-bdp
+destination:
+  platform: aws_s3
+  path: "observations/reanalysis/omps/nasa/ompsnm_eff/%Y/%m/netcdf/"
```
Comment (frolovsa) [inline data-tree/reanalysis/omps/nasa/2012-2021.ompsnm_eff.clean_bucket.yaml]: reanalysis/ozone/nasa/omps-nm-eff

File: data-tree/reanalysis/omps/nasa/2013-2021.ompsnp.clean_bucket.yaml [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+    platform: aws_s3
+    key:  observations/scratch_from_jrk/ompsnp/Y%Y/M%m/
+    bucket: noaa-reanalyses-pds
+    file_template:
+      prefix: "OMPSNP.%Y%m%d_%Hz"
+      suffix: ".nc"
+    credential_profile: rnr-bdp
+destination:
+  platform: aws_s3
+  path: "observations/reanalysis/omps/nasa/ompsnp/%Y/%m/netcdf/"
```
Comment (frolovsa) [inline data-tree/reanalysis/omps/nasa/2013-2021.ompsnp.clean_bucket.yaml]: reanalysis/ozone/nasa/omps-nm

File: data-tree/reanalysis/sbuv/README.md [hunk 9]
```diff
@@ -0,0 +1 @@
+archive of sbuv data
```
Comment (frolovsa) [inline data-tree/reanalysis/sbuv/README.md]: reanalysis/ozone/nasa/sbuv_v87/

File: data-tree/reanalysis/sbuv/nasa/1999-2018.clean_bucket.yaml [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+    platform: aws_s3
+    key:  observations/scratch_from_jrk/sbuv_v87/bufr/Y%Y/M%m/
+    bucket: noaa-reanalyses-pds
+    file_template:
+      prefix: "sbuv_v87.%Y%m%d.%Hz."
+      suffix: ".bufr"
+    credential_profile: rnr-bdp
+destination:
+  platform: aws_s3
+  path: "observations/reanalysis/sbuv/nasa/sbuv_v87/%Y/%m/bufr/"
```
Comment (frolovsa) [inline data-tree/reanalysis/sbuv/nasa/1999-2018.clean_bucket.yaml]: reanalysis/ozone/nasa/sbuv_v87/

**Review by frolovsa — CHANGES_REQUESTED** [review]
Thank you, Jessica,

I made some suggestions to the paths to collocate all the ozone under the same top-level directory. 

I did not check the start-end dates. I will rely on you to do that. 
Also, if you want you can set `remove source file: true` if you want to automatically delete the temp files. Otherwise you will need to  remove them by hand alter.

File: data-tree/reanalysis/omi/nasa/2004-2020.clean_bucket.yaml (hunk 4 above)
Comment (jrknezha) [reply data-tree/reanalysis/omi/nasa/2004-2020.clean_bucket.yaml]: thanks for sharing, interesting read and I'll keep learning about the different data streams

File: data-tree/reanalysis/mls/nasa/2004-2021.clean_bucket.yaml (hunk 2 above)
Comment (jrknezha) [reply data-tree/reanalysis/mls/nasa/2004-2021.clean_bucket.yaml]: updated

File: data-tree/reanalysis/omps/README.md (hunk 5 above)
Comment (jrknezha) [reply data-tree/reanalysis/omps/README.md]: updated

File: data-tree/reanalysis/omps/nasa/2012-2021.ompslp.clean_bucket.yaml (hunk 6 above)
Comment (jrknezha) [reply data-tree/reanalysis/omps/nasa/2012-2021.ompslp.clean_bucket.yaml]: updated

File: data-tree/reanalysis/omps/nasa/2012-2021.ompsnm_eff.clean_bucket.yaml (hunk 7 above)
Comment (jrknezha) [reply data-tree/reanalysis/omps/nasa/2012-2021.ompsnm_eff.clean_bucket.yaml]: updated

File: data-tree/reanalysis/omps/nasa/2013-2021.ompsnp.clean_bucket.yaml (hunk 8 above)
Comment (jrknezha) [reply data-tree/reanalysis/omps/nasa/2013-2021.ompsnp.clean_bucket.yaml]: updated

File: data-tree/reanalysis/sbuv/README.md (hunk 9 above)
Comment (jrknezha) [reply data-tree/reanalysis/sbuv/README.md]: updated

File: data-tree/reanalysis/sbuv/nasa/1999-2018.clean_bucket.yaml (hunk 10 above)
Comment (jrknezha) [reply data-tree/reanalysis/sbuv/nasa/1999-2018.clean_bucket.yaml]: updated

**Comment by jrknezha** [comment]
reorganized the structure of the files in the data-tree to reflect grouping in ozone and made appropriate changes to each file for that structure

**Review by frolovsa — APPROVED** [review]
looks good

**Pull request merged**